spec/views/fields/edit_spec.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-include RSpecHtmlMatchers
 
 describe 'fields/edit.haml' do
+  include RSpecHtmlMatchers
```

Comment:
So, Rubocop is right. This is where you should include per-spec helpers. But I can't help but notice that it's included in _all_ the view specs.

Is there a way to just say that in the spec helper?

```ruby
config.include RSpecHtmlMatchers, type: :view
```

Reply:
#96 if you think so